packages/cli/test/integration/shared/utils.ts

```diff
@@ -161,17 +223,17 @@ export async function initTestServer({
 /**
  * Pre-requisite: Mock the telemetry module before calling.
  */
-export function initTestTelemetry() {
-	void InternalHooksManager.init('test-instance-id', mockNodeTypes);
+export async function initTestTelemetry() {
```

Comment:
Now called at a single spot so a separate function might not be needed, as was done with `initTestLogger`.